### DISCUSSION TITLE
I can't access my main GitHub account where my account is associated different email

### DISCUSSION THREAD

**Discussion by saipirakib** [community/community#45945] [New to GitHub] [answered]
I can't access my main GitHub account where my account is associated with two important emails of mine. Please help me to recover this. please help me to solve this issue

**Comment by queenofcorgis** [comment, marked as answer]
Hi @saipirakib ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them, I'll be locking this post as they are the best route for help :)
